FlowCrypt/Controllers/SideMenu/Menu/MyMenuViewController.swift

```diff
@@ -50,7 +50,9 @@ final class MyMenuViewController: ASDKViewController<ASDisplayNode> {
 
     private var folders: [FolderViewModel] = []
     private var serviceItems: [FolderViewModel] { FolderViewModel.menuItems }
-    private var accounts: [User] { appContext.dataService.validAccounts() }
+    private var accounts: [User] {
+        appContext.dataService.getFinishedSetupUsers(exceptUserEmail: currentUser.email)
```

Comment:
Here we use that filter.

Reply:
Should be called `otherAccounts` :-)

Reply:
And if it's only used here, then we should just do the filtering here instead of an argument